src/api/claimReviewTask.ts

```diff
@@ -0,0 +1,42 @@
+import { message } from "antd";
+import axios from "axios";
+
+const request = axios.create({
+    withCredentials: true,
+    baseURL: `/api/claimreviewtask`,
+});
+
+const createClaimReviewTask = (params, t) => {
+    return request
+        .post('/', {...params})
+        .then((response) => {
+            message.success(t("claimReviewTask:userAssignedSuccess"))
+            return response.data
+        })
+        .catch(err=>{
+            message.error(t("claimReviewTask:userAssignedError"))
+            return err
+        })
+}
+
+const updateClaimReviewTask = (params, t) => {
+    console.log(params)
+    return request
+        .put(
+            `/${params.sentence_hash}`,
+            { ...params },
+            { withCredentials: true }
```

Comment:
Not needed because the axios instance is already setting this parameter for the requests in this file